Importer/EventListener/InterpreterExceptionSubscriber.php

```diff
@@ -50,6 +51,8 @@ private function resolveInterpreterExceptionMessage(InterpreterException $except
             return "Row is expected to contain {$exception->getExpectedSize()}, found to have {$exception->getFoundSize()} fields instead.";
         } elseif ($exception instanceof InvalidRowException) {
             return $this->buildInvalidRowExceptionMessage($exception);
+        } elseif ($exception instanceof BooleanFormatException) {
```

Comment:
Well we have a generic exception. It's the `InterpreterException`. The reason that we have other types of exceptions is because of the information we want to show to user when something goes wrong.